index.html

```diff
@@ -7,6 +7,32 @@
     <title>Personal Portfolio</title>
 </head>
 <body>
-    <p>Haregebezu Amare</p>
+    <h1>Haregebezu Amare</h1>
+    <h2>About</h2>
+    <p>I have immersed myself in the world of software development for over a year. With an insatiable curiosity and passion
+    for technology, I have delved into various programming languages, frameworks, and tools, building a solid foundation of
+    technical knowledge. From mastering web development with HTML, CSS, and JavaScript, they have developed a versatile
+    skill set. I am committed to lifelong learning, staying up to date with industry trends through workshops and online
+    courses. Each day, embrace the challenges and rewards of being a software developer, poised to make a significant impact
+    in the world of technology. </p>
+    <h2> Experience</h2>
+    <p> During my virtual classes in front-end development, I had the opportunity to acquire a comprehensive set of skills and
+    deepen my understanding of the field. Over the course of the program, I delved into HTML, CSS, and JavaScript, gaining
+    proficiency in crafting responsive and visually appealing user interfaces. The virtual class provided a platform for
+    hands-on learning, where I successfully completed various projects that challenged me to implement complex layouts,
+    optimize website performance, and integrate interactive features. Collaborating with classmates and instructors in the
+    virtual environment allowed me to sharpen my communication and teamwork skills, as we leveraged online forums and video
+    conferences to exchange ideas and provide valuable feedback. I encountered and overcame numerous obstacles throughout
+    the program, developing strong problem-solving abilities and an adaptive mindset. By continuously seeking additional
+    resources beyond the virtual classes, such as online tutorials and coding challenges, I demonstrated my commitment to
+    lifelong learning and staying updated in this rapidly evolving field. Overall, the virtual class experience in front-end
+    development has equipped me with the knowledge, skills, and confidence to pursue real-world projects and thrive in the
+    industry.</p>
+    <h2>Connect</h2>
+    <li><a href="mailto:your-email@example.com" target="_blank">Haregebezuamare@gmail.com </a></li>
+
+    <li><a href="https://github.com/Haregebezu"> GitHub </a></li>
+
+    <li><a href="https://www.linkedin.com/in/haregebezu-amare/"> Linkedin </a></li>
```

Comment:
```suggestion
    <ul>
      <li><a href="mailto:your-email@example.com" target="_blank">Haregebezuamare@gmail.com</a></li>
      <li><a href="https://github.com/Haregebezu">GitHub</a></li>
      <li><a href="https://www.linkedin.com/in/haregebezu-amare/">Linkedin</a></li>
    </ul>
```

`<li>` should be inside `<ul>` or `<ol>`. Also watch out for extra spaces inside your tags.